src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -26,8 +26,14 @@
                 class="form-control"
                 rows="2"
             ></textarea>
+            <div v-if="isFeatureLine">
+                <font-awesome-icon :icon="['far', 'square']" /> {{ length }}
+            </div>
+            <div v-if="isFeaturePolygon">
+                <font-awesome-icon :icon="['fas', 'square']" style="color: #888a85" /> {{ area
+                }}<sup>2</sup>
```

Comment:
try to break the line after the `>` from FontAwesomeIcon, it's correct linter wise how it's done here, but it is not really easy to read